uaclient/tests/test_security.py

```diff
@@ -1522,6 +1522,7 @@ def test_messages_for_affected_packages_when_fix_fail(
         out, err = capsys.readouterr()
         assert expected in out
 
+    @pytest.mark.parametrize("should_reboot", (False, True))
```

Comment:
@lucasmoura I added this so to ensure this test worked whether should_reboot is False or True